api-spec/protobuf/tdexa/v1/analytics.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package tdexa.v1;
+
+/**
+ * Analytics service spec
+ */
+service Analytics {
+  // returns all markets and its balances in time series
+  rpc MarketsBalances(MarketsBalancesRequest) returns (MarketsBalancesReply) {}
+  // returns all markets and its prices in time series
+  rpc MarketsPrices(MarketsPricesRequest) returns (MarketsPricesReply) {}
+}
+
```

Comment:
One parameter I would always add is the time "window" ie.  `last hour`, `last day`, `last month`, `year to date`, `all`

Reply:
should we keep for client side to calculate fromTime? api user can easily pass wanted value(last hour, day etc)

Reply:
I think its just semantic, bit more user firendly to have clients to tell us via an enum.

The `fromTime` is implicit, ie. lastHours means the window from now and hour before.

Maybe can be useful for PRO/advanced users? but at that point let them specify a time range [from,to]

Reply:
@tiero can we postpone this through new issue?